gordo_components/builder/build_model.py

```diff
@@ -217,10 +219,7 @@ def _score_per_tag(y_true, y_pred):
             )
 
         metrics_dict.update(
-            {
-                metric_str.replace("_", "-")
-                + f"-tag-mean": make_scorer(metric_wrapper(metric))
-            }
+            {metric_str.replace("_", "-"): make_scorer(metric_wrapper(metric))}
```

Comment:
I think everywhere you use `metric_str` you do this `.replace(...)` maybe just do that at the assignment 
`metric_str = metric.__name__.replace('_', '-')`